modulefiles/module_gwci.hera.lua

```diff
@@ -0,0 +1,15 @@
+help([[
+Load environment to run GFS workflow setup scripts on Hera
+]])
+
+prepend_path("MODULEPATH", "/scratch2/NCEPDEV/nwprod/hpc-stack/libs/hpc-stack/modulefiles/stack")
+
+load(pathJoin("hpc", "1.1.0"))
+load(pathJoin("hpc-intel", "18.0.5.274"))
+load(pathJoin("hpc-impi", "2018.0.4"))
+
+load(pathJoin("netcdf","4.7.0"))
```

Comment:
```suggestion
load(pathJoin("netcdf","4.7.4"))
```

This is the version used by workflow.

Reply:
Pretty sure you had this fixed at one point and now it has reverted back.